<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceDlg</class>
 <widget class="QDialog" name="DeviceDlg">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>261</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>581</width>
    <height>261</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>581</width>
    <height>261</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="pbDo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Doen</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRoutine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Routine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuleer</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbRoutine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Maak een nieuwe routine</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbDevice">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Voeg een onderdeel toe</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblClass">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Onderdeel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDeclare">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblDeclare">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Declaratie</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblInclude">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Include</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leInclude">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblUsing">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Using</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbClass">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbUsing">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbVariable">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Maak een nieuwe variabele</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTypeVar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTypeVar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbRoutineDel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwijder een routine</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbVariableDel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwijder een variabele</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbDeviceDel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwijder een onderdeel</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbRoutine">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbVariable">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblVariable">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Variabele</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDevice">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblDevice">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Naam</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRoutine">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leVariable">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDevice">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbTypeDev">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblTypeDev">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pbDo</zorder>
  <zorder>lblRoutine</zorder>
  <zorder>pbCancel</zorder>
  <zorder>rbRoutine</zorder>
  <zorder>rbDevice</zorder>
  <zorder>lblClass</zorder>
  <zorder>leDeclare</zorder>
  <zorder>lblDeclare</zorder>
  <zorder>lblInclude</zorder>
  <zorder>leInclude</zorder>
  <zorder>lblUsing</zorder>
  <zorder>cbClass</zorder>
  <zorder>cbUsing</zorder>
  <zorder>rbVariable</zorder>
  <zorder>lblTypeVar</zorder>
  <zorder>cbTypeVar</zorder>
  <zorder>line_4</zorder>
  <zorder>rbRoutineDel</zorder>
  <zorder>rbVariableDel</zorder>
  <zorder>rbDeviceDel</zorder>
  <zorder>line_5</zorder>
  <zorder>line_6</zorder>
  <zorder>cbRoutine</zorder>
  <zorder>cbVariable</zorder>
  <zorder>lblVariable</zorder>
  <zorder>cbDevice</zorder>
  <zorder>lblDevice</zorder>
  <zorder>leRoutine</zorder>
  <zorder>leVariable</zorder>
  <zorder>leDevice</zorder>
  <zorder>cbTypeDev</zorder>
  <zorder>lblTypeDev</zorder>
 </widget>
 <tabstops>
  <tabstop>rbRoutine</tabstop>
  <tabstop>rbRoutineDel</tabstop>
  <tabstop>leRoutine</tabstop>
  <tabstop>cbRoutine</tabstop>
  <tabstop>rbVariable</tabstop>
  <tabstop>rbVariableDel</tabstop>
  <tabstop>leVariable</tabstop>
  <tabstop>cbVariable</tabstop>
  <tabstop>cbTypeVar</tabstop>
  <tabstop>rbDevice</tabstop>
  <tabstop>rbDeviceDel</tabstop>
  <tabstop>leDevice</tabstop>
  <tabstop>cbDevice</tabstop>
  <tabstop>cbTypeDev</tabstop>
  <tabstop>cbClass</tabstop>
  <tabstop>cbUsing</tabstop>
  <tabstop>leInclude</tabstop>
  <tabstop>leDeclare</tabstop>
  <tabstop>pbDo</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
